examples/end-to-end-testing/flake.lock

```diff
@@ -0,0 +1,128 @@
+{
+  "nodes": {
```

Comment:
We can omit this lockfile, none of the other examples do (I'm not entirely sure if Nix will blindly copy it when used as a template or not)

Reply:
Oops, how did that end up in here?

> I'm not entirely sure if Nix will blindly copy it when used as a template or not

Had to check, Yep the lockfile gets copied.